cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -15,11 +15,14 @@ use self::cincinnati::plugins::prelude_plugin_impl::*;
 use tokio::sync::Mutex as FuturesMutex;
 
 pub static DEFAULT_OUTPUT_ALLOWLIST: &[&str] = &[
-    "/LICENSE$",
-    "/channels/.+\\.ya+ml$",
-    "/blocked-edges/.+\\.ya+ml$",
-    "/raw/metadata.json$",
-    "/version$",
+    // The patterns may be potentially matched against different relative
+    // paths depending on the underlying decoder or the specific archive.
+    // for example: `archive/version` or `version`
+    "(^|/)LICENSE$",
+    "(^|/)channels/.+\\.ya+ml$",
+    "(^|/)blocked-edges/.+\\.ya+ml$",
+    "(^|/)raw/metadata.json$",
+    "(^|/)version$",
```

Comment:
i'm particularly concerned about this change. 
we've similar code fragment in the other secondary parser https://github.com/openshift/cincinnati/blob/master/cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper/plugin.rs#L13-L19

does this change affect the outcome of e2es etc ?

Reply:
When debugging the unit tests I have written down my findings in the https://github.com/openshift/cincinnati/pull/978/commits/005098f95304f59398b677d4e4b86e6433f7e325 commit:

> ...
Currently, the regex patterns inside the
`dkrv2_openshift_secondary_metadata_scraper` plugin are matched
against the archive files (e.g. `version`); however, regex
patterns inside the `github_openshift_secondary_metadata_scraper`
plugin are matched against the files prefixed with the archive
directory (e.g. `archive/version`). This is most likely dependent
on the underlying decoder library or the specific archive.
...

I did not have much luck in understanding why. I have concluded that the chosen decoder or the archive likely influences the functionality.

Let me run the CI on the PR changes excluding this one to check e2e :+1: Without this change, the unit tests locally did not pass for me as nothing was getting matched by them.

Reply:
The [c466179](https://github.com/openshift/cincinnati/commit/c4661796468ae71fd6bf773a475e123a72eb31c6) in the  [full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift&repo=cincinnati&pr=978) reverted the commits regarding the modifications to the `DEFAULT_OUTPUT_ALLOWLIST` variable.

I am dropping the reverts.